test/e2e/e2eutil.go

```diff
@@ -127,7 +127,7 @@ func NewTestMongoDB(ctx *Context, name string, namespace string) (mdbv1.MongoDBC
 										Name: "mongodb-agent",
 										Resources: corev1.ResourceRequirements{
 											Limits: map[corev1.ResourceName]resource.Quantity{
-												"cpu":    resource.MustParse("0.1"),
+												"cpu":    resource.MustParse("1.0"),
```

Comment:
Quite a bit of change! Is it required?

Reply:
This is only in e2e tests, so I didn't give it much thought. 
When run in GHA and on local non M1 mac this was sufficient. Unfortunately on macs there is additional QEMU layer, so everything is less performant and with 0.1 limit is just hanging.
This change shouldn't be invasive, i.e. shouldn't cause pods to be unschedulable as it's only limit, not request.

Reply:
ooh yes, I'm sorry, it's limits and I'm blind!